extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1187,7 +1191,7 @@ export class AmpA4A extends AMP.BaseElement {
           // the keys, probably b/c of a network condition.  We should
           // re-trigger key fetching later.
           return {serviceName: currServiceName, keys: []};
-        });
+        }));
```

Comment:
Nit: You can move the close-paren to the line with `jwkSetObj`, which results in less nesting overall.